demos/config_solo12.yaml

```diff
@@ -4,36 +4,37 @@ robot:
         motor_numbers: [0, 3, 2, 1, 5, 4, 6, 9, 8, 7, 11, 10]
         motor_constants: 0.025
         gear_ratios: 9.
-        max_currents: 1.
+        max_currents: 8.
         reverse_polarities: [
-            true, false, true, true, false, false,
-            true, false, true, true, false, false
+            false, true, true, true, false, false,
+            false, true, true, true, false, false
```

Comment:
On our solo I have to use
```
true, true, true, false, false, false,
true, true, true, false, false, false
```
to be consistent with the urdf we are using (https://github.com/Gepetto/example-robot-data/tree/master/robots/solo_description/robots/solo12.urdf). To check that I loaded the robot using its urdf in a viewer (gepetto-viewer-corba) then I continuously set its configuration to robot.joints.positions. While sending 0 torques, I moved the legs by hand and verified that the urdf robot in the viewer moved in the same way (if the polarity is wrong the legs in the viewer move in the other direction than in reality).

I'm not sure what could explain our difference, maybe a urdf change or a different wiring on the robot ?

Reply:
The motor numbers are the same for you? I will verify the urdf you linked in a bit. Also, I will do a live visualization and double check again.

Reply:
Can you verify the wiring of solo? We use what we have specified on the wiki for the wiring order.